static/app/utils/replays/replayReader.tsx

```diff
@@ -493,6 +493,13 @@ export default class ReplayReader {
 
   getRRWebFrames = () => this._sortedRRWebEvents;
 
+  getRRWebFramesWithoutMediaInteractions = () =>
+    this._sortedRRWebEvents.filter(
+      ({type, data}) =>
+        type === EventType.IncrementalSnapshot &&
+        data.source !== IncrementalSource.MediaInteraction
+    );
+
```

Comment:
I might actually filter this in replayStepper instead since it's pretty specific to the stepper than consumers of the reader.